doc/src/cylc-user-guide/cug.tex

```diff
@@ -1257,12 +1257,12 @@ \subsubsection{Configuring Cylc Review}
 Alternatively you can run the Cylc Review web service under Apache
 \lstinline=mod_wsgi=. To do this you will need to set up an Apache module
 configuration file (typically in
-\lstinline=/etc/httpd/conf.d/rose-wsgi.conf=) containing the following (with
+\lstinline=/etc/httpd/conf.d/cylc-wsgi.conf=) containing the following (with
 the paths set appropriately):
 
 \lstset{language=bash}
 \begin{lstlisting}
-WSGIPythonPath /path/to/rose/lib/python
+WSGIPythonPath /path/to/cylc/lib/python
```

Comment:
This might be confusing.  Rose has, literally, a `lib/python` sub-dir, but Cylc doesn't.  I think we're referring to the cylc install location elsewhere in the docs as `<cylc-dir>` ... so this should probably be `<cylc-dir>/lib/cylc` (location of `review.py`).

@kinow - this raises a question!  On your packaging branch, with other bundled packages removed, we probably don't need `<cylc-dir>/lib/cylc` anymore ... should we standardise to `<cylc-dir>/lib/python/`?  And e.g. move `job.sh` from current `lib/cylc` to `lib/bash`?

Reply:
Oh, you are right @hjoliver ! Maybe it should be 

```
WSGIPythonPath /path/to/cylc/lib/
```

?

>On your packaging branch, with other bundled packages removed, we probably don't need <cylc-dir>/lib/cylc anymore ... should we standardise to <cylc-dir>/lib/python/? And e.g. move job.sh from current lib/cylc to lib/bash?

Not sure to be honest. I think the Melbourne conf will come in right time for it. I have a few points and questions about how to implement the packaging, and also what must be done (or avoided).

I think the answer for the first question would be yes, we can move from `lib/cylc` to `lib/python`, or to just `cylc`, or any other folder (though we would still have the `parsec` module to move too right now).

For the second question, I think we could do that. I had to add to the PR for packaging an option to include the `job.sh` file with the other Python scripts. But `setup.py` is very malleable. It requires you to provide a few values for those `setuptools` options... but then you are also free to write any Python code you'd like to versioning, packaging, bundling, copying files around, etc. Only risk is creating a too complex process.

Not quite confident in change the whole packaging yet, but in a quick brainstorm session we should be able to clear these points, and remove the `WIP` from that branch! 🎉 🚀